docker/package/model.py

```diff
@@ -367,13 +367,13 @@ def fetch_sources(self, out_dir):
             ]
         )
 
-    def gen_control_file(self, deps, out):
+    def gen_control_file(self, deps, ubuntu_version, out):
         file_contents = f"""
 Source: {self.name}
 Section: utils
 Priority: optional
 Maintainer: {self.meta.maintainer}
-Build-Depends: debhelper (>=9), dh-systemd (>= 1.5), autotools-dev, wget
+Build-Depends: debhelper (>=9), autotools-dev, wget
```

Comment:
You pass the `ubuntu_version` parameter but never use it here :thinking: Did you mean to do the same thing as in other two changed places and add the conditional usage clause?

Reply:
oh, indeed, thanks for catching